api/app/config.py

```diff
@@ -42,8 +41,9 @@ def get_kubeseal_version() -> str:
 
 
 def get_app_config():
-    """Return the app configs in json format."""
-    config = {}
-    config["kubeseal_version"] = get_kubeseal_version()
+    """Return the app configs dict."""
+    config = {
```

Comment:
We could return the dict directly instead of storing it in a temporary variable.